build.gradle

```diff
@@ -165,11 +182,83 @@ task downloadAppEngineSDK() {
   }
 }
 
+task gaeClean() {
+  delete "${buildDir}/gae"
+}
+
+task gaeCopyFiles(dependsOn: explodedWar, type: Copy) {
+  into "${buildDir}/gae/ear"
+  from fileTree(dir: 'src/main/gae_ear')
+
+  into("background") {
+    from fileTree(dir: "${buildDir}/exploded")
+    exclude "WEB-INF/classes/odk-settings.xml"
+    exclude "WEB-INF/classes/security.properties"
+  }
+
+  into("default") {
+    from fileTree(dir: "${buildDir}/exploded")
+    exclude "WEB-INF/classes/odk-settings.xml"
+    exclude "WEB-INF/classes/security.properties"
+  }
+
+  doLast {
+    delete "default/WEB-INF/lib/guava-jdk5-17.0.jar"
+    delete "background/WEB-INF/lib/guava-jdk5-17.0.jar"
+  }
+}
+
+task gaeCopySDK(dependsOn: gaeDownloadSDK, type: Copy) {
+  into "${buildDir}/gae"
+  from zipTree("${projectDir}/installer/appEngineSDK.zip")
+}
+
+task gaeCopyDummyApp(type: Copy) {
+  into "${buildDir}/gae/dummyApp"
+  from fileTree(dir: 'installer/dummyApp')
+}
+
+task gaePrepare(dependsOn: [gaeCopyFiles, gaeCopySDK, gaeCopyDummyApp]) {
```

Comment:
You also need to set `security.server.realm.realmString`. It should probably be `${aggregateInstanceName} ODK Aggregate"`